scripts/test_nearlib.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -ex
+
+./target/release/devnet &
+./scripts/waitonserver.sh
+./scripts/build_wasm.sh
+
+# Run nearlib tests
+cd nearlib
+npm install
+npm run build
+npm run doc
+npm test
+cd ..
+
+# Try creating and building new project using NEAR CLI tools
+rm -rf new_project
+mkdir new_project
+cd new_project
+npm install near-shell
+node_modules/near-shell/near new_project
+npm install
+npm run build
+npm test
+cd ..
+
+./scripts/kill_devnet.sh
```

Comment:
kill_devnet.sh is not working. no need to include it unless we fix it :)
It also breaks CI if you fix it

Reply:
oh lol when did it get broken? Also tests used to fail if devnet was not killed, so it's surprising

Reply:
I don't think it ever worked since it doesn't have run env specified, unless it guesses it from the parent script.

Reply:
@evgenykuzyakov what is this then?

```
+ ./scripts/kill_devnet.sh
./scripts/test_nearlib.sh: line 27:    11 Killed                  ./target/release/devnet
```

https://gitlab.com/nearprotocol/nearcore/-/jobs/184784258

Reply:
*magic*